features/step_definitions/machine_set.rb

```diff
@@ -4,3 +4,38 @@
   cb_name ||= :machinesets
   cb[cb_name] = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
 end
+
+Given(/^I pick a random machineset to scale$/) do
+  machine_sets = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
+  cache_resources *machine_sets
+
+  @machine_set = machine_sets.shuffle.first
```

Comment:
If you want to get a random machine set, then just cache like:
```
cache_resources *machine_sets.shuffle
```

Later just call method `machine_set` method to get the random one.